### PR TITLE
Replacing --export flag

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/7124

### CODE REVIEW

**Review by dislbenn — APPROVED** [review]
LGTM

**Comment by songleo** [comment]
hi @dockerymick , I saw your PR and found there are some issues, for step 3, should be the following:

3. Generate your pull-secret. If Red Hat Advanced Cluster Management is installed in the `open-cluster-management` namespace, run the following command to generate your secret:

   ```
   DOCKER_CONFIG_JSON=`oc extract secret/multiclusterhub-operator-pull-secret -n open-cluster-management --to=-`
   oc create secret generic multiclusterhub-operator-pull-secret \
       -n open-cluster-management-observability \
       --from-literal=.dockerconfigjson="$DOCKER_CONFIG_JSON" \
       --type=kubernetes.io/dockerconfigjson
   ```

   If the `multiclusterhub-operator-pull-secret` is not defined in the `open-cluster-management` namespace, copy the `pull-secret` from the `openshift-config` namespace into the `open-cluster-management-observability` namespace. Run the following command:

   ```
   DOCKER_CONFIG_JSON=`oc extract secret/pull-secret -n openshift-config --to=-`
   oc create secret generic multiclusterhub-operator-pull-secret \
       -n open-cluster-management-observability \
       --from-literal=.dockerconfigjson="$DOCKER_CONFIG_JSON" \
       --type=kubernetes.io/dockerconfigjson
   ```

you can refer to this doc to update it: https://github.com/open-cluster-management/multicluster-monitoring-operator#installation  thanks.

File: observability/observability_install.adoc [hunk 1]
```diff
@@ -28,16 +28,15 @@ oc create namespace open-cluster-management-observability
 
 +
 ----
-oc get secret multiclusterhub-operator-pull-secret -n open-cluster-management --export -o yaml |  kubectl apply --namespace=open-cluster-management-observability -f -
+oc extract secret/multiclusterhub-operator-pull-secret -n open-cluster-management --to=-
```
Comment (songleo) [inline observability/observability_install.adoc:31]: the cmd should like this:

```
DOCKER_CONFIG_JSON=`oc extract secret/multiclusterhub-operator-pull-secret -n open-cluster-management --to=-`
oc create secret generic multiclusterhub-operator-pull-secret \
    -n open-cluster-management-observability \
    --from-literal=.dockerconfigjson="$DOCKER_CONFIG_JSON" \
    --type=kubernetes.io/dockerconfigjson
```

File: observability/observability_install.adoc [hunk 2]
```diff
@@ -28,16 +28,15 @@ oc create namespace open-cluster-management-observability
...
 ----
 +
 If the `multiclusterhub-operator-pull-secret` is not defined in the namespace, copy the pull-secret from the `openshift-config` namespace into the `open-cluster-management-observability` namespace. Run the following command:
 +
 ----
-oc get secret pull-secret -n openshift-config --export -o yaml | sed 's/name:\ pull-secret/name: multiclusterhub-operator-pull-secret/' | kubectl apply --namespace=open-cluster-management-observability -f -
+oc create secret generic multiclusterhub-operator-pull-secret -n open-cluster-management-observability --from-literal=.dockerconfigjson="$DOCKER_CONFIG_JSON" --type=kubernetes.io/dockerconfigjson
```
Comment (songleo) [inline observability/observability_install.adoc:37]: the cmd should like this:

```
   DOCKER_CONFIG_JSON=`oc extract secret/pull-secret -n openshift-config --to=-`
   oc create secret generic multiclusterhub-operator-pull-secret \
       -n open-cluster-management-observability \
       --from-literal=.dockerconfigjson="$DOCKER_CONFIG_JSON" \
       --type=kubernetes.io/dockerconfigjson
```

**Comment by dockerymick** [comment]
@songleo update made with this PR: https://github.com/open-cluster-management/rhacm-docs/pull/1548/files

**Comment by songleo** [comment]
@dockerymick thank you so much.

**Pull request merged**